src/tools/check_brokenURL.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+
+import re
+import os
+import sys
+import getopt
+import requests
+import argparse
+
+def get_filepaths(directory):
+    file_paths = []  
+    for root, directories, files in os.walk(directory):
+        for filename in files:
+            filepath = os.path.join(root, filename)
+            file_paths.append(filepath)
+    return file_paths
+
+def checkBrokenLink(url):
```

Comment:
We can try to exclude that from the list of urls to be checked. 
I will add this it exclude everything that follows https://github.com/ceph